tslint.json

```diff
@@ -1,11 +1,8 @@
 {
-    "rulesDirectory": ["tslint-microsoft-contrib"],
```

Comment:
So this is also intentional change related to package.json npm audit? I think if this is removed how do we take care of linting ? because each person's linting depends on their machine and it might mess up. so we need to think about it if we are removing tslint rules.

Reply:
Exactly, thanks for this, yeah, this was done carefully so that I can only do upgrade and fix vulnerability under this PR and possibly not make this a `tslint` to `eslint` migration PR. :) ☕️

Once packages are updated, then we can add the `tslint` to `eslint` migration PR which could or might not be small. But let's definitely remove the old outdated packages, before we get critical dependentbot warning.